packages/transformers/js/src/JSTransformer.js

```diff
@@ -180,9 +180,14 @@ export default new Transformer({
       res.code = generated.code;
 
       let map = null;
-      if (generated.rawMappings) {
+      if (generated.map) {
         map = new SourceMap();
-        map.addIndexedMappings(generated.rawMappings);
+        // $FlowFixMe flow-types are incomplete
+        map.addRawMappings(
+          generated.map.mappings,
```

Comment:
Encoding VLQ and the decoding it again is faster than sending objects? Strange...

Reply:
Damn just realised my benchmark is still using some old code, will update to see if it still makes sense...

Reply:
It still does seem faster
![Screenshot 2020-03-07 at 19 36 40](https://user-images.githubusercontent.com/2175521/76150446-ff2bf880-60aa-11ea-97b7-425cbea083ff.png)